config/runtime.exs

```diff
@@ -0,0 +1,168 @@
+    game_name: "Beyond All Reason",
+    game_name_short: "BAR",
+    main_website: "https://www.beyondallreason.info/",
+    privacy_email: "privacy@beyondallreason.info",
+    discord: "https://discord.gg/beyond-all-reason",
+    ports: [
+      tcp: Teiserver.ConfigHelpers.get_env("TEI_SPRING_TCP_PORT", 8200, :int),
+      tls: Teiserver.ConfigHelpers.get_env("TEI_SPRING_TLS_PORT", 8201, :int),
+      # this can likely be deprecated and removed. It's for an old version
+      # of tachyon running on another TLS socket
+      tachyon: Teiserver.ConfigHelpers.get_env("TEI_TACHYON_TLS_PORT", 8202, :int)
+    ],
+    certs: certificates,
+    website: [
+      url: "beyondallreason.info"
+    ],
+    server_flag: "GB-WLS",
+    enable_benchmark: false,
+    node_name: Teiserver.ConfigHelpers.get_env("TEI_NODE_NAME"),
+    extra_logging: false,
+    enable_managed_lobbies: true,
+    user_agreement:
+      "A verification code has been sent to your email address. Please read our terms of service at https://#{domain_name}/privacy_policy and the code of conduct at https://www.beyondallreason.info/code-of-conduct. Then enter your six digit code below if you agree to the terms."
+
+  config :teiserver, Teiserver.Repo,
+    hostname: Teiserver.ConfigHelpers.get_env("TEI_DB_HOSTNAME"),
+    username: Teiserver.ConfigHelpers.get_env("TEI_DB_USERNAME"),
+    password: Teiserver.ConfigHelpers.get_env("TEI_DB_PASSWORD"),
+    database: Teiserver.ConfigHelpers.get_env("TEI_DB_NAME"),
+    pool_size: 40,
+    timeout: 120_000,
+    queue_interval: 2000
+
+  check_origin =
+    if Teiserver.ConfigHelpers.get_env("TEI_SHOULD_CHECK_ORIGIN", false, :bool) do
+      ["//#{domain_name}", "//*.#{domain_name}"]
+    else
+      false
+    end
+
+  config :teiserver, TeiserverWeb.Endpoint,
+    url: [host: domain_name],
+    check_origin: check_origin,
+    https:
+      certificates ++
+        [
+          versions: [:"tlsv1.2"],
+          # dhfile is not supported for tls 1.3
+          # https://www.erlang.org/doc/man/ssl.html#type-dh_file
+          dhfile: Teiserver.ConfigHelpers.get_env("TEI_TLS_DH_FILE_PATH", "/etc/ssl/dhparam.pem")
+        ],
+    http: [:inet6, port: Teiserver.ConfigHelpers.get_env("TEI_PORT", "4000", :int)],
+    secret_key_base: Teiserver.ConfigHelpers.get_env("TEI_HTTP_SECRET_KEY_BASE")
+
+  config :teiserver, Teiserver.Account.Guardian,
+    issuer: Teiserver.ConfigHelpers.get_env("TEI_GUARDIAN_ISSUER", "teiserver"),
+    secret_key: Teiserver.ConfigHelpers.get_env("TEI_GUARDIAN_SECRET_KEY")
+
+  if Teiserver.ConfigHelpers.get_env("TEI_ENABLE_EMAIL_INTEGRATION", :bool) do
+    config :teiserver, Teiserver.Mailer,
+      adapter: Bamboo.SMTPAdapter,
+      contact_address:
+        Teiserver.ConfigHelpers.get_env("TEI_CONTACT_EMAIL_ADDRESS", "info@#{domain_name}"),
+      noreply_name: "Beyond All Reason",
+      noreply_address:
+        Teiserver.ConfigHelpers.get_env("NOREPLY_EMAIL_ADDRESS", "noreply@#{domain_name}"),
```

Comment:
Missing `TEI_` prefix... took me embarrassingly long time to find because the email wasn't delivered only between the SMTP server and my real email...

Reply:
Fixed